src/MessagingAPI.ts

```diff
@@ -24,6 +24,9 @@ export interface MessagingAPI {
     /** Get total number of unseen messages from all conversations the user has joined */
     getTotalUnseenMessages(): number
 
+    getProfileInfo(userId: string): Promise<{ displayName?: string; avatarUrl?: string }>
+
+    getMemberInfo(roomId: string, userId: string): { displayName?: string; avatarUrl?: string }
```

Comment:
And I think this one too:
```suggestion
    getMemberInfo(roomId: string, userId: string): ProfileInfo
```